docs/embedding/static-embedding-parameters.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Parameters for static embeds
 redirect_from:
-- /docs/latest/embedding/signed-embedding-parameters
+  - /docs/latest/embedding/signed-embedding-parameters
```

Comment:
Is this whitespace intentional? Seems unrelated to the changes

Reply:
I formatted the doc with `prettier`.